src/Exceptions/User/CreateUserErrorException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Exceptions\User;
+
+class CreateUserErrorException extends \Exception
```

Comment:
Create a more specific exception based on \LogicException

```
namespace App\Exception\User;
abstract class UserWhateverFailed extends \LogicException
```

Also I can't find where they are thrown 🤔The namespace should be descriptive of the source IMO